boards.txt

```diff
@@ -691,3 +691,88 @@ d1_mini.menu.FlashSize.4M1M.build.spiffs_start=0x300000
 d1_mini.menu.FlashSize.4M1M.build.spiffs_end=0x3FB000
 d1_mini.menu.FlashSize.4M1M.build.spiffs_blocksize=8192
 d1_mini.menu.FlashSize.4M1M.build.spiffs_pagesize=256
+
+
+##############################################################
+espino.name=ESPino
+
+espino.upload.tool=esptool
+espino.upload.speed=115200
+espino.upload.resetmethod=ck
+espino.upload.maximum_size=1044464
+espino.upload.maximum_data_size=81920
+espino.upload.wait_for_upload_port=true
+espino.serial.disableDTR=true
+espino.serial.disableRTS=true
+
+espino.build.mcu=esp8266
+espino.build.f_cpu=80000000L
+espino.build.board=ESP8266_ESP12
+espino.build.core=esp8266
+espino.build.variant=espino
+espino.build.flash_mode=qio
+espino.build.flash_size=4M
+espino.build.flash_freq=40
+espino.build.spiffs_pagesize=256
+
+espino.menu.UploadTool.esptool=Serial
+espino.menu.UploadTool.esptool.upload.tool=esptool
+espino.menu.UploadTool.esptool.upload.verbose=-vv
+espino.menu.UploadTool.espota=OTA
+espino.menu.UploadTool.espota.upload.tool=espota
+
+espino.menu.CpuFrequency.80=80 MHz
+espino.menu.CpuFrequency.80.build.f_cpu=80000000L
+espino.menu.CpuFrequency.160=160 MHz
+espino.menu.CpuFrequency.160.build.f_cpu=160000000L
+
+espino.menu.FlashFreq.40=40MHz
+espino.menu.FlashFreq.40.build.flash_freq=40
+espino.menu.FlashFreq.80=80MHz
+espino.menu.FlashFreq.80.build.flash_freq=80
+
+espino.menu.FlashMode.dio=DIO
```

Comment:
Since you probably know the exact flash model used by this board, DIO/QIO and flash frequency selection is probably unnecessary?